app/views/charges/new.html.erb

```diff
@@ -1,4 +1,15 @@
-<%= form_tag charges_path do %>
-  <h4>Click the button!</h4>
-  <script class='stripe-button' src="https://checkout.stripe.com/checkout.js" data-key="<%= @stripe_btn_data[:key] %>" data-amount=<%= @stripe_btn_data[:amount] %> data-description="<%= @stripe_btn_data[:description] %>" ></script>
-<% end %>
+<div class="container">
+  <% if current_user.premium? %>
+    <%= button_to 'Downgrade Membership', downgrade_path, class: 'btn' %>
```

Comment:
@bjblock - How do I call the `downgrade` action within the `user_controller` here? I'm running into the issue of the user's membership being downgrading anytime I visit `localhost:3000/charges/new` which isn't obviously what we want here. I'm struggling to implement it within the button alone.